### PR TITLE
SA: rollback transaction on exc

### PR DESCRIPTION
**Pull request by TomasTomecek**
Is this a better way of doing sqlalch?

Fixes RED-HAT-0P-2AH

### CODE REVIEW

File: packit_service/models.py [hunk 1]
```diff
@@ -161,16 +168,10 @@ class CoprBuild(Base):
     data = Column(JSON)
 
     def set_status(self, status: str):
-        session = get_sa_session()
         self.status = status
-        session.add(self)
-        session.commit()
```
Comment (dhodovsk) [inline packit_service/models.py]: How is this going to propagate into database?

Comment (TomasTomecek) [reply packit_service/models.py]: `autocommit=True`

every time you change a SA object, it gets automatically committed to the database; see the tests :) which means we no longer need to do `.commit()`

Comment (dhodovsk) [reply packit_service/models.py]: Neat!

**Comment by sakalosj** [comment]
LGTM

**Pull request merged**